products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusTimerPresenter.cpp

```diff
@@ -102,7 +113,22 @@ namespace app::focus
     {
         timer->stop();
         betweenSessionTimer.stop();
-        app->switchWindow(focus::window::name::main);
+
+        const auto elapsed = std::chrono::duration_cast<std::chrono::minutes>(timer->getElapsed());
+        const auto minutesInFocus =
+            elapsed.count() + (allFocusSessionsCount - focusSessionsLeft) * focusSessionDuration.count();
+        const auto sumOfFocusTime =
+            std::to_string(minutesInFocus) + " " + utils::language::getCorrectMinutesAccusativeForm(minutesInFocus);
+        const auto textToComplete = utils::translate(app_focus_timer_summary);
+        const auto summaryText    = createSummaryText(textToComplete, sumOfFocusTime);
```

Comment:
If I see correctly, the text "app_focus_timer_summary" is translated twice. The first time on line 122 and the second time inside the "createSummaryText" function